.github/scripts/prepare-quarkus-next.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+git checkout -b new-quarkus-next origin/main
+
+if ! grep -q '<repositories>' pom.xml; then
+  sed -i '/<\/project>/i \
+          <repositories> \
+          </repositories>' pom.xml
+fi
+
+# Insert the <repository> element before the closing </repositories> tag
+sed -i '/<\/repositories>/i \
+        <repository> \
+        <id>sonatype-snapshots</id> \
+        <name>Sonatype Snapshots</name> \
+        <url>https://s01.oss.sonatype.org/content/repositories/snapshots/</url> \
+        <snapshots> \
+            <enabled>true</enabled> \
+            <updatePolicy>daily</updatePolicy> \
+        </snapshots> \
+        <releases> \
+            <enabled>false</enabled> \
+        </releases> \
+        </repository>' pom.xml
+
+./mvnw versions:set-property -Dproperty=quarkus.version -DnewVersion=999-SNAPSHOT
+./mvnw versions:set-property -Dproperty=quarkus.build.version -DnewVersion=999-SNAPSHOT
```

Comment:
@Pepo48 could a maven profile be used to encapsulate this?  The additional repository and quarkus versions seems like it could be handled there.

Reply:
@shawkins I tried in one of the earlier iterations (I believe it was based on your first review). I remember that based on an offline feedback from Vašek/Stian it was decided to not introduce another profile just because the changes in this PR.

Reply:
@Pepo48 ok, so we're forced into pom manipulation to add a repository then. One last thought - the sed / set-version could be handled in the MAVEN_ARGS env variable - MAVEN_ARGS="-Dquarkus.version=999-SNAPSHOT ..."

Reply:
Based on an offline discussion with @Pepo48 and @shawkins, we keep the current approach.